public/ACMW/css/events.css

```diff
@@ -51,7 +42,7 @@
 	color: #d5d5d5;
 }
 .navbar-nav li a:hover {
-	color: #fff;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)

Reply:
here too

Reply:
Have a look at this: [Disallow !important](https://github.com/CSSLint/csslint/wiki/Disallow-!important)